src/Sylius/Bundle/AdminBundle/Resources/views/Order/show.html.twig

```diff
@@ -47,6 +47,7 @@
 
         {% include '@SyliusAdmin/Order/Show/_payments.html.twig' %}
         {% include '@SyliusAdmin/Order/Show/_shipments.html.twig' %}
+        {% include '@SyliusAdmin/Order/Show/_resendEmail.html.twig' %}
```

Comment:
It is just a personal filling, but I would say, that it would be better to add this button next to cancel buttons etc at the top of the page

Reply:
It is quite a big button and I've consulted that with @CoderMaggie that this will be a proper place